hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -338,8 +338,8 @@ public OmKeyInfo lookupKey(OmKeyArgs args, String clientAddress)
         bucketName);
 
     BucketLayout bucketLayout =
-        getBucketLayout(metadataManager, args.getVolumeName(),
-            args.getBucketName());
+        OzoneManagerUtils.getBucketLayout(args.getVolumeName(),
+            args.getBucketName(), ozoneManager, new HashSet<>());
```

Comment:
I have minor code suggestions for this refactoring:

1. Create another `getBucketLayout` method (in `OzoneManagerUtils`) without the `Set<Pair<String, String>> visited` parameter, passing `new HashSet<>()` to the existing `getBucketLayout` method.
2. Make `OzoneManager` the first param of this new method.
3. Add static import for `OzoneManagerUtils.getBucketLayout`.

This would make the code more readable and minimize the changes necessary to satisfy line width rules.

Reply:
Thanks a lot for the reviews @adoroszlai, I have addressed these points in my latest patch.
I had a typo in one place which seemed to be causing the CI errors, waiting on the new run to verify the behavior now.